gallery/workbook/microsoft.network-azurefirewalls.json

```diff
@@ -7,16 +7,21 @@
 			"name": "Network Insights AzureFirewalls",
 			"templates": [
 				{
-					"id": "Workbooks/Network Insights/AzureFirewallWorkbooks/NetworkInsights-AzureFirewallMetrics-Minified",
+					"id": "Workbooks/Network Insights/AzureFirewallWorkbooks/MinifiedWorkbook",
```

Comment:
i wouldn't move/rename the ids of workbooks, thse are ids, they might be in places opening them in code, depending on these in links/pins, etc.

my only comment as on the NAMES displayed to users.

Reply:
Updated